autorag_client/tutorials/.env.example

```diff
@@ -0,0 +1,2 @@
+AUTORAG_API_KEY=ar-kkhDPdtr1VciePOQR-N7Y_
```

Comment:
The real api key seems unsafe. Is it real api key????

Reply:
It is not real key